components/user/UserPreview.js

```diff
@@ -21,7 +22,7 @@ export default function UserPreview({ profile }) {
           />
         </div>
         <h3 className="text-xl font-bold md:hidden">
-          {profile.name} {profile.views && <span>({profile.views})</span>}
+          {profile.name} {abbreviateNumber(profile.views) && <span>({abbreviateNumber(profile.views)})</span>}
```

Comment:
I don't think the first `views` needs the formatting

```suggestion
          {profile.name} profile.views && <span>({abbreviateNumber(profile.views)})</span>}
```